src/lib/addEventListener.js

```diff
@@ -13,4 +12,4 @@ function addEventListener(eventName, handler, el) {
   }
 }
 
-module.exports = IS_CLIENT ? addEventListener : noop;
+export default IS_CLIENT ? addEventListener : noop;
```

Comment:
This will break if we ever switch from a transpiler to a theoretical native ES6 module system, since all `import` and `export` calls must be static. I'd just do the `IS_CLIENT` check inside the `addEventListener` function itself.